src/main/java/com/teragrep/cfe18/handlers/CaptureGroupController.java

```diff
@@ -84,96 +84,75 @@ public class CaptureGroupController {
     CaptureGroupMapper captureGroupMapper;
 
     @RequestMapping(
-            path = "/group/{name}",
-            method = RequestMethod.GET,
+            path = "",
```

Comment:
let's change api path to
```
/v2/captures/group/$groupId/members
```
where put results to 
```
/v2/captures/group/$groupId/members/$captureId
```

and delete to that removes the id from the group
get on the members lists all members